multimedia/libvpx/Portfile

```diff
@@ -72,6 +72,10 @@ configure.args      --enable-vp8 \
                     --disable-examples \
                     --disable-unit-tests
 
+if { ${build_arch} eq "arm64" || [variant_isset universal] } {
+    configure.args-delete --enable-runtime-cpu-detect
+}
```

Comment:
Ok, now you've changed it so that `--enable-runtime-cpu-detect` is removed for a non-universal arm64 build and for a universal build regardless of the archs. Is that what you meant to do?

I had no problem building libvpx universal x86_64/i386 on High Sierra, presumably with runtime CPU detection. If you're now proposing that runtime CPU detection needs to be disabled for that build, then you need to increase the port's revision.

Why does runtime CPU detection need to be disabled at all?

Reply:
No, it is only needed for arm64. I was just unsure how to limit the change to arm64 under universal--and unaware people are still doing universal builds on x86. I've pushed another guess on how to do this correctly (I reenable the flag using `merger_configure_args`). If you have some quick advice or documentation links for how to do this more elegantly I would be happy to learn. I wasn't able to find much by googling.

Runtime detection is not implemented yet for arm on macOS (a compiler error is produced if we request it). The current implementations are for Windows and Linux (parsing `/proc/cpuinfo`) only, and are not portable. However, the compile-time detection of NEON features looks completely fine for arm Macs.

Reply:
There's not a lot of documentation about runtime cpu detection in libvpx, but presumably it allows a single binary to be used, e.g., on a system implementing AVX2 and one with only AVX, while still running the optimal supported codepath on both. Since all arm Macs support the same NEON instructions (for now) this is not currently needed.